plugins/mission/include/plugins/mission/mission_item.h

```diff
@@ -52,6 +52,13 @@ class MissionItem {
      */
     void set_fly_through(bool fly_through);
 
+    /**
+     * @brief Set the acceptance radius property of a mission item.
+     *
+     * @param radius Radius around the mission_item where it will be considered as reached
+     */
+    void set_acceptance_radius(float radius);
```

Comment:
I would add the unit to be consistent with the others in the file (e.g. `altitude_m`).

```suggestion
    void set_acceptance_radius(float radius_m);
```